cypress/support/stripes.js

```diff
@@ -0,0 +1,26 @@
+const DEFAULT_SEARCH_PARAMS = {
+  limit: 1000,
+  query: 'cql.allRecords=1',
+};
+
+Cypress.Commands.add('okapiRequest', ({
+  method = 'GET',
+  path,
+  searchParams = {},
+  body,
+}) => {
+  const queryString = (new URLSearchParams({
+    ...DEFAULT_SEARCH_PARAMS,
+    ...searchParams,
+  })).toString();
+
+  cy.request({
+    method,
+    url: `${Cypress.env('OKAPI_HOST')}/${path}?${queryString}`,
```

Comment:
From where OKAPI_HOST variable is going to be taken?

Reply:
https://wiki.folio.org/pages/viewpage.action?pageId=62555156
configuration section